### PR TITLE
CIV-10296 wording fixed

### PR DESCRIPTION
**Pull request by dtortolaV1**
### JIRA link (if applicable) ###
[CIV-10296](https://tools.hmcts.net/jira/browse/CIV-10296)

### Change description ###
Wording change

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by MMNycz — APPROVED** [review]

**Review by indu2019 — DISMISSED** [review]

**Review by asthamalviya — APPROVED** [review]

**Review by suvarnathekkoot — COMMENTED** [review]
Is the change required for both 1v1 and 1v2?

**Comment by dtortolaV1** [comment]
> Is the change required for both 1v1 and 1v2?

No, for 1v1 and 2v1

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -33,4 +33,8 @@
 
   </suppress>
 
+  <suppress until="2023-10-04">
```
Comment (indu2019) [inline config/owasp/suppressions.xml:36]: just check if its not the same CVE, which sampan asked not to add.

**Review by indu2019 — DISMISSED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (sabahirfan) [reply config/owasp/suppressions.xml:36]: This is fixed by https://github.com/hmcts/civil-service/pull/3184, please remove this spression.

**Review by asthamalviya — APPROVED** [review]

**Review by MMNycz — APPROVED** [review]

**Review by indu2019 — APPROVED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (dtortolaV1) [reply config/owasp/suppressions.xml]: removed

**Pull request merged**